recipe/meta.yaml

```diff
@@ -20,6 +20,7 @@ requirements:
   build:
     - m4              # [unix]
     - python
+    - toolchain       # [unix]
```

Comment:
It might shave a little build time and hopefully better compartmentalize things. Doesn't have to be done in this PR or anything. Just a suggestion that we try it at some point.